ec2/minimal/README.md

```diff
@@ -0,0 +1,122 @@
+# CATalogASB EC-2 Deployment
```

Comment:
Can we change this line to match the convention used above? CatASB instead of CATalogASB?

Reply:
yes... will update